app/views/devise/registrations/new.html.erb

```diff
@@ -0,0 +1,49 @@
+<html>
+  <head>
+    <%= stylesheet_link_tag 'user_new' %>
+  </head>
+
+  <body>
+    <div class=classContainer>
+      <h2>Sign up</h2>
+
+      <%= form_for(resource, as: resource_name, url: registration_path(resource_name)) do |f| %>
+        <%= render "devise/shared/error_messages", resource: resource %>
+
+        <div class="field">
+          <%= label :user, :name %>
+          <%= text_field :user, :name, autofocus: true, class:'input'  %>
+        </div>
+        
+          <div class="field">
+            <%= label :user, :bio %>
+            <%= text_field :user, :bio, autofocus: true, class:'input'  %>
+          </div>
+
+        <div class="field">
+          <%= f.label :email %><br />
+          <%= f.email_field :email, autofocus: true, autocomplete: "email", class:'input'  %>
+        </div>
+
+        <div class="field">
+          <%= f.label :password %>
+          <% if @minimum_password_length %>
+          <em>(<%= @minimum_password_length %> characters minimum)</em>
+          <% end %><br />
+          <%= f.password_field :password, autocomplete: "new-password", class:'input'  %>
+        </div>
+
+        <div class="field">
+          <%= f.label :password_confirmation %><br />
+          <%= f.password_field :password_confirmation, autocomplete: "new-password", class:'input'  %>
+        </div>
+
+        <div class="actions">
+          <%= f.submit "Sign up", class:'btn' %>
```

Comment:
- [ ] Similarly, [as suggested](https://github.com/smunoz1988/blog-on-rails/pull/9#discussion_r1276786351) for signup, please make sure to use placeholders instead of labels. Although the wireframe for signup is not given, the signup page should be consistent with the given wireframe ✨📌

Current design | Wireframe
---| ---|
![image](https://github.com/smunoz1988/blog-on-rails/assets/106119105/e43ce47f-b8c2-4774-be36-5e8b68b03411) | ![image](https://github.com/smunoz1988/blog-on-rails/assets/106119105/c7e4e9ae-2446-4a29-a310-fecc1a5feb6b)